### PR TITLE
Rebrand the site generator.

### PR DESCRIPTION
**Pull request by rofrankel**
All instances of aip (and variants) have been replaced with the equivalent form of aep.

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -1,2 +1,2 @@
-# The AIP infrastructure team manages this repository.
-*   @aip-dev/infra
+# The AEP infrastructure team manages this repository.
```
Comment (earth2marsh) [inline .github/CODEOWNERS:1]: We don't have an @aep-dev/infra team, but perhaps we should create one? Or maybe we can change this to the maintainers team for now?

**Review by earth2marsh — APPROVED** [review]
Just one note about the CODEOWNERS file, but that doesn't need to hold up merging! Thanks for making these changes.

File: .github/CODEOWNERS (hunk 1 above)
Comment (earth2marsh) [reply .github/CODEOWNERS:1]: I've added the team, and we can figure out the membership later.

Comment (rofrankel) [reply .github/CODEOWNERS:1]: Good catch, and thanks!

**Pull request merged**